packages/react-core/src/components/TextArea/TextArea.tsx

```diff
@@ -8,7 +8,8 @@ import { FormControlIcon } from '../FormControl/FormControlIcon';
 export enum TextAreResizeOrientation {
   horizontal = 'horizontal',
   vertical = 'vertical',
-  both = 'both'
+  both = 'both',
+  disabled = 'disabled'
```

Comment:
Can't recall if there was a conversation regarding this, but was using "none" considered instead of "disabled", to match the CSS value that would normally be used? Not a blocker since "disabled" is still accurate, just wondering if people may think a modifier class will be applied like in other instances odf a disabled modifier.

Reply:
I actually debated between the two names. I started with `none`.  I can change it back.